frontend/src/workbench/Workbench.types.ts

```diff
@@ -48,7 +48,7 @@ export interface WorkbenchViewContributionProps {
   side: WorkbenchViewSide;
   title: string;
   icon: React.ReactElement;
-  component: (props: WorkbenchViewComponentProps) => JSX.Element;
+  component: (props: WorkbenchViewComponentProps) => React.ReactElement | null;
```

Comment:
`JSX.Element | null` seems to be working so I'll stick with that and I'll update my PR with this. The main issue is that a React component can return `null` and it was not taken into account in the signature. Since the types in questions were not all properly exported, Sirius Web could thus have code which would technically not compile.

As for the shift, this is the existing behavior. You could try it on the test instance to confirm it.

Reply:
I've updated the commit with the type `JSX.Element | null`

Reply:
For the shift: I did try on the test instance and did not see it there, but re-testing it actually depends on the size of the left panel relative to the width of the edition field.